content/talks/responsive-components.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Container Queries & The Future of CSS
+sub: Modernizing the cascade for responsive design systems
+author: miriam
+image:
+  src: talks/container-query.jpg
+  width: 1279
+  height: 696
+  position: top
+tags:
+  - Cascade
+  - CSS
+  - Container Queries
+  - CSS Scope
+  - Cascade Layers
+  - CSSWG
+events:
+  - venue: Web Directions, Hover
+    url: https://webdirections.org/hover/
+    date: 2021-04-09
+    end: 2021-04-30
+    adr: Online
+  - venue: Modern Web Conference
+    url: https://modernwebconference.com/
+    date: 2021-05-17
+    end: 2021-05-28
+    adr: Online
+summary: |
+  New CSS proposals like Container Queries,
+  Cascade Layers, Scoped Styles, and Nesting
+  are all aimed at improving the way we write
+  responsive components and design systems.
+---
+
+Over the last decade
+Object-Oriented & Responsive design
+have become the norm --
+with tools like Flexbox, Grid,
+intrinsic sizing, and `aspect-ratio`
+giving us even more layout control.
+CSS has always been designed for a responsive web,
+but that goalpost can shift over time.
+These new features will help bring CSS up-to-speed
+with the latest patterns in modern web design.
+Let's see how they work!
```

Comment:
After reading this a few times, it may be helpful to weave in what you will teach the attendee which I think can be done with something as simple as adding:

...  "Join Miriam for this session to learn how these new features will help bring CSS up-to-speed with the latest .... "

Reply:
@stacyk how's that?

Reply:
@mirisuzanne perfect!